src/components/Table/index.less

```diff
@@ -1,8 +1,7 @@
-tr > td > span > a {
-  text-decoration: none !important;
-  color: inherit !important;
-}
-
 .table-wrapper {
   overflow-x: auto;
 }
+
+tr:is(.pf-m-hoverable):hover {
+  background: aliceblue;
+}
```

Comment:
@aquibbaig I am still a little reluctant about adding a complete different colour on the hover action. Instead how about using PatternFly's provided default value `var(--pf-c-table--tr--m-hoverable--active--BackgroundColor)` ?

Also, I think it would be great to make a little bit more customisation to make it look like  PF table. How about recreating the [style of odd rows](var(--pf-c-table--tr--m-hoverable--active--BackgroundColor)) for active/ hoverable rows?

Reply:
@anishaswain Got point regarding reusing the Patternfly color variable, let me add that to the code. Also, can you share some designs for the suggested customization? The above link shows a 404 for me

Reply:
@aquibbaig I have updated the link

Reply:
Just a heads up @anishaswain - the link you shared seems to point to an old release of PatternFly 3. We'll want to make sure we reference documentation from PatternFly 4's latest release.

Reply:
@gurbirkalsi the link points to `2020.04` documentation. The link is `https://www.patternfly.org/2020.04/documentation/react/components/table#custom-row-wrapper`. Let me know if I missed something here.

Reply:
The latest release should be `2021.08` at [https://www.patternfly.org/v4/components/table#custom-row-wrapper](url).